Alignment/CommonAlignmentProducer/plugins/GlobalTrackerMuonAlignment.cc

```diff
@@ -600,27 +629,21 @@ void GlobalTrackerMuonAlignment::analyzeTrackTrack(const edm::Event& iEvent, con
     std::cout << std::endl;
```

Comment:
there are std::cout in the modified files, it is advised to change them to the edm::LogVerbatim